changelog.d/4802.misc

```diff
@@ -0,0 +1 @@
+ Switch from simplejson to the standard library json.
```

Comment:
This matches #7800, seemed silly to have a separate changelog entry for changing a comment. :)